manim/mobject/svg/text_mobject.py

```diff
@@ -803,6 +803,9 @@ def __init__(
             **kwargs,
         )
         self.text = text
+        if self.disable_ligatures:
+            self.submobjects = [*self.gen_chars()]
```

Comment:
self.submojects is not being set when self.disable_ligatures is False. Or am I missing anything :thinking:

Reply:
That should be set iirc, initialising is done by something else.